bot/exts/evergreen/snakes/_snakes_cog.py

```diff
@@ -311,12 +311,12 @@ async def _get_snek(self, name: str) -> Dict[str, Any]:
 
         async with aiohttp.ClientSession() as session:
```

Comment:
Should use the bot's `http_session`

Reply:
Resolved in https://github.com/python-discord/sir-lancebot/pull/718/commits/780aaeb5c272f7a9fd9fe9e8cdc1c041beb90a25.